src/JWTGuard.php

```diff
@@ -74,9 +74,14 @@ public function user()
 
         if ($this->jwt->getToken() && $this->jwt->check()) {
             $id = $this->jwt->payload()->get('sub');
```

Comment:
It's useless to resolve `$id` outside `if ($this->provider->getModel() == $provider) {`